pixart.js

```diff
@@ -1 +1,37 @@
+$(document).ready(function(){
 
+var buttonClicked = $("#set-color");
+
+function changeColor(evt){
+  var colorSelected=$("#color-field").val();
+  evt.preventDefault();
+  $("div .brush").css("background-color",colorSelected);
+  console.log("color changed to "+ colorSelected);
+  $("body").on("click",squareCreated);
+}
+
+buttonClicked.on("click",changeColor);
+
+$("#color-field").on("keypress",enterKeyPressed);
+function enterKeyPressed(e){
+  if (e==13)
```

Comment:
`e` will not equal `13` since it is an object. You need to look for the right property within the `e` object.